modules/ocf/manifests/packages/paper_icons.pp

```diff
@@ -0,0 +1,9 @@
+class ocf::packages::paper_icons {
```

Comment:
I believe this whole class, along with the one to set up the apt repo/key can be removed now since @abizer uploaded the `paper-icon-theme` package into our internal repo, so the package can just be specified in the list where you put `arc-theme`.